deploy/helm_charts/envs/unsdg_staging.yaml

```diff
@@ -67,7 +67,7 @@ kgStoreConfig:
     project: datcom-un-staging
     instance: dc-graph
     tables:
-      - ilo_2024_06_24_09_46_00
-      - sdg_2024_06_25_10_21_43
+      - ilo_2024_07_02_14_25_51
```

Comment:
To confirm, this is assuming you will use "deploy_latest.sh" to deploy, since the latest country version may not be checked into bigtable versions yaml, right?

Reply:
That's right